File: NVIM/nvim/lua/USER/modules/playground/file.py
def calculadora():
    num1 = float(input("Ingrese el primer número: "))
    num2 = float(input("Ingrese el segundo número: "))
    operacion = input("Ingrese la operación (+, -, *, /): ")

    if operacion == "+":
        resultado = num1 + num2
    elif operacion == "-":
        resultado = num1 - num2
    elif operacion == "*":
        resultado = num1 * num2
    elif operacion == "/":
        if num2 == 0:
            print("Error: Division por cero")
            return
        resultado = num1 / num2
    else:
        print("Error: Operación inválida")
        return

    print(num1, operacion, num2, "=", resultado)

calculadora()

File: Nvim/nvim/lua/USER/modules/lsp/playground/file.php
<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    $num1 = floatval($_POST["num1"]);
    $num2 = floatval($_POST["num2"]);
    $operacion = $_POST["operacion"];

    switch ($operacion) {
        case '+':
            $resultado = $num1 + $num2;
            break;
        case '-':
            $resultado = $num1 - $num2;
            break;
        case '*':
            $resultado = $num1 * $num2;
            break;
        case '/':
            if ($num2 == 0) {
                echo "Error: Division por cero";
                return;
            }
            $resultado = $num1 / $num2;
            break;
        default:
        echo "Error: Operación inválida";
        return;
    }

    echo $num1, " ", $operacion, " ", $num2, " = ", $resultado;
}
?>
